lib/determine-basal/determine-basal.js

```diff
@@ -566,22 +566,22 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
             remainingCIs.push(round(remainingCI,0));
             predCIs.push(round(predCI,0));
             //process.stderr.write(round(predCI,1)+"+"+round(remainingCI,1)+" ");
-            COBpredBG = COBpredBGs[COBpredBGs.length-1] + predBGI + Math.min(0,predDev) + predCI + remainingCI;
-            aCOBpredBG = aCOBpredBGs[aCOBpredBGs.length-1] + predBGI + Math.min(0,predDev) + predACI;
+            var COBpredBG = COBpredBGs[COBpredBGs.length-1] + predBGI + Math.min(0,predDev) + predCI + remainingCI;
```

Comment:
Related to comment below, determine-basal crashes with this declaration:
```
/usr/local/lib/node_modules/oref0/lib/determine-basal/determine-basal.js:697
        avgPredBG = round( (1-fractionCarbsLeft)*UAMpredBG + fractionCarbsLeft*COBpredBG );
                                                                               ^

ReferenceError: COBpredBG is not defined
    at determine_basal (/usr/local/lib/node_modules/oref0/lib/determine-basal/determine-basal.js:697:80)
    at Object.<anonymous> (/usr/local/lib/node_modules/oref0/bin/oref0-determine-basal.js:215:14)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3
No deliverAt found.
```
Removing it fixes the issue.